README.md

```diff
@@ -5,6 +5,13 @@
 
 This is the repository of the W3C’s specification on JSON-LD 1.1 Processing Algorithms and API, developed by the [JSON-LD Working Group](https://www.w3.org/2018/json-ld-wg/). The editors’ draft of the specification can also be [read directly](https://w3c.github.io/json-ld-api/).
 
+## Feature Frozen
+
+We are no longer accepting new features for JSON-LD 1.1 drafts. Please limit
+requests to bugs and clarifications of the existing feature set.
```

Comment:
requests -> issues

Reply:
fixed in latest, but with a slightly more verbose text to avoid the hard to read "issues to bugs" combo.